src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ShippingMethod.xml

```diff
@@ -23,7 +23,7 @@
         </attribute>
 
         <attribute name="normalization_context">
-            <attribute name="groups">admin:shipping_method:read</attribute>
+            <attribute name="groups">shop:shipping_method:read</attribute>
```

Comment:
Wouldn't this lead to show shop normalization for admin_put, delete and archive? Maybe deleting this block would be better and declare normalization context where it is needed?